Updating MCP docs based on feedback (#8137)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-ai/about-mcp.md

> ## Server Access

Reviewer: ```suggestion
## Server access
```

> The dbt MCP server can be used in two ways: locally or remotely. Choose the setup that best fits your workflow:

Reviewer: ```suggestion
You can use the dbt MCP server in two ways: locally or remotely. Choose the setup that best fits your workflow:
```

### Local MCP server

The local MCP server provides the best experience for development workflows, like authoring dbt models, tests, and documentation.

The [local MCP server](/docs/dbt-ai/setup-local-mcp) runs on your machine and requires installing `uvx` (which installs dbt-mcp locally). This option provides:
- Full access to dbt CLI commands (`dbt run`, `dbt build`, `dbt test`, and more)
- Support for <Constant name="core" />, <Constant name="cloud_cli" />, and <Constant name="fusion_engine" />
- Ability to work with local dbt projects without requiring a <Constant name="dbt_platform" /> account
- Optional integration with <Constant name="dbt_platform" /> APIs for metadata discovery and Semantic Layer access

### Remote MCP server

The remote MCP server from dbt offers data consumption use cases without local setup.

The [remote MCP server](/docs/dbt-ai/setup-remote-mcp) connects to the <Constant name="dbt_platform" /> via HTTP and requires no local installation. This option is useful when:

> - You don't want to or are restricted from installing additional software on your system.

Reviewer: ```suggestion
- You either don’t want to install, or are restricted from installing, additional software on your system.
```

> - Your use case is primarily consumption-based (querying metrics, exploring metadata, viewing lineage).

Reviewer: ```suggestion
- Your use case is primarily consumption-based (for example, querying metrics, exploring metadata, viewing lineage).
```

> ## Available Tools

Reviewer: ```suggestion
## Available tools
```

## Server Access

> The dbt MCP server can be installed in two ways: locally or remotely. Choose the setup that best fits your workflow:

Reviewer 2: ```suggestion
The dbt MCP server can be used in two ways: locally or remotely. Choose the setup that best fits your workflow:
```

### Local MCP server

> **Best for:** Full development workflow including build and manage.

Author: ```suggestion
The local MCP server from dbt offers full development workflows, including build and manage.
```

> The local MCP server from dbt offers full development workflows, including build and manage.

Reviewer 2: ```suggestion
The local MCP server provides the best experience for development workflows, like authoring dbt models, tests, and documentation.
```

The [local MCP server](/docs/dbt-ai/setup-local-mcp) runs on your machine and requires installing `uvx` (which installs dbt-mcp locally). This option provides:
- Full access to dbt CLI commands (`dbt run`, `dbt build`, `dbt test`, and more)
- Support for <Constant name="core" />, <Constant name="cloud_cli" />, and <Constant name="fusion_engine" />
- Ability to work with local dbt projects without requiring a <Constant name="dbt_platform" /> account

> - Optional integration with <Constant name="dbt_platform" /> APIs for metadata discovery and Semantic Layer access

Reviewer 2: Nice!

:::tip Configuring local MCP
To disable tools that call remote APIs (Discovery API, Semantic Layer API) if you're operating exclusively within an IDE or working with <Constant name="core" /> only. You can disable these tools using environment variables like `DISABLE_DISCOVERY=true` and `DISABLE_SEMANTIC_LAYER=true`.

> :::

Reviewer 2: Our latest changes make this no longer necessary. Now, dbt Platform features are only enabled when the  `DBT_HOST` environment variable is present. So, to disable those tools, the user needs to not set that environment variable.

Similarly, if a user is just interested in dbt Platform features and not CLI features, they can just not set the `DBT_PROJECT_DIR` and `DBT_PATH` environment variables.

### Remote MCP server

> **Best for:** Consumption and management use cases without local setup.

Author: ```suggestion
The remote MCP server from dbt offers both consumption and management use cases without local setup.
```

> The remote MCP server from dbt offers both consumption and management use cases without local setup.

Reviewer 2: `consumption` and `management` are likely too ambiguous. Mainly, we use those terms internally. `management` is coming soon, when we host the admin api tools.

```suggestion
The remote MCP server from dbt offers both data consumption use cases without local setup.
```

Author: Just goingt to remove the "both" language then
```suggestion
The remote MCP server from dbt offers data consumption use cases without local setup.
```

The [remote MCP server](/docs/dbt-ai/setup-remote-mcp) connects to the <Constant name="dbt_platform" /> via HTTP and requires no local installation. This option is useful when:
- You don't want to or are restricted from installing additional software on your system.
- Your use case is primarily consumption-based (querying metrics, exploring metadata, viewing lineage).

> - You want to manage dbt platform resources (trigger jobs, view runs, access artifacts).

Reviewer 2: Can't do this one yet. The admin api for remote MCP is coming soon.

> - You want to manage dbt platform resources (trigger jobs, view runs, access artifacts).

Author: ```suggestion
```

## Troubleshooting

### Local MCP setup issues

> #### Can't find `uvx` executable

Reviewer 2: This section is much improved! Should we move it to the local MCP page since it only applies to that and not the remote MCP?
